web/src/app/schedules/calendar-subscribe/formHelper.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core'
+import { CheckCircleOutline as SuccessIcon } from '@material-ui/icons'
+import CalenderSuccessForm from './CalendarSuccessForm'
+
+const useStyles = makeStyles(theme => ({
+  successIcon: {
+    marginRight: theme.spacing(1),
+  },
+  successTitle: {
+    color: 'green',
+    display: 'flex',
+    alignItems: 'center',
+  },
+}))
+
+export function FormTitle(isComplete, defaultTitle) {
```

Comment:
Is this worth breaking out into it's own file? also `FormTitle` doesn't match the file name and is a bit generic for something specific to calendar subscriptions

Reply:
^ meaning is `formHelper.js` as a whole worth being it's own file

Reply:
We can if we really want to, though the only reason I have it named as a component rather than a normal function is because of my linter yelling at me (likely because I'm returning a node). I like them being bundled together because they're 3 small functions to aid in rendering the proper form

Reply:
Are they re-used across multiple files though? If it's worth it to be it's own file I suggest naming the file after the component, and name the component as the others `CalendarSubscribeFormTitle` or something like that. Quick to go from dev-tools -> file without digging around that way.

Reply:
It originally was, but later changed to only be referenced in the one file because we no longer need the success form in the Edit dialog, so good point.